scripts/initialize.sh

```diff
@@ -33,7 +36,11 @@ help()
    echo -e "\t--beneficiary: Staking beneficiary address"
    echo -e "\t--authorizer: Staking authorizer address"
    echo -e "\t--stake-amount: Staking amount"
-   echo -e "\t--authorization-amount: Authorization amount\n"
+   echo -e "\t--authorization-amount: Authorization amount"
+   echo -e "\t--tbtc-path: 'Local' tbtc project's path. A temporary folder with tbtc is created and removed"\
+                           "upon installation if the path is not provided"
+   echo -e "\t--skip-ecdsa-deployment: This option skips ecdsa and tbtc deployment. Default is false"
+   echo -e "\t--skip-tbtc-deployment: This option skips tbtc deployment. Default is false\n"
```

Comment:
I don't follow, why do we care about tbtc in this script?

Reply:
Pls see https://github.com/keep-network/keep-core/pull/3096#discussion_r931015776 & https://github.com/keep-network/keep-core/pull/3096#discussion_r931026710